pkg/utils/utils.go

```diff
@@ -236,3 +242,42 @@ func DeleteDoubleWhiteSpace(slice []string) []string {
 	}
 	return result
 }
+
+// PopulateNamespacesForResources populates namespaces from config file
+func PopulateNamespacesForResources(include, ignore []string) []string {
```

Comment:
Can you please unit tests for this func ?

Reply:
resolved